compiler/ballerina-parser/src/test/resources/misc/identifiers/keywords_as_identifiers_source.bal

```diff
@@ -1,19 +1,19 @@
-type LoopContext record {|
-    LoopContext? outer;
-    string str;
-    boolean ascending;
-    boolean descending;
-|};
+//type LoopContext record {|
+//    LoopContext? outer;
+//    string str;
+//    boolean ascending;
+//    boolean descending;
+//|};
```

Comment:
Instead of commenting out inside the test. Shall we disable the test with `@Test(enabled = false)`, otherwise we will forget this as a diabled test in the future.

Reply:
Did you mean to extract the valid test to another test and disable this test?

Reply:
Didn't notice the valid part. It's okay then.

Reply:
+1